criterion-external.cabal

```diff
@@ -30,19 +28,19 @@ executable criterion-interactive
                        deepseq
```

Comment:
Similarly, put a lower bound here.

Reply:
Good catch! I also bumped the Stackage resolver to `10.7`, which has `criterion-1.2.6.0`.